chia/cmds/wallet.py

```diff
@@ -99,18 +99,19 @@ def show_cmd(wallet_rpc_port: int, fingerprint: int) -> None:
     asyncio.run(execute_with_wallet(wallet_rpc_port, fingerprint, {}, print_balances))
 
 
-@wallet_cmd.command("get_address", short_help="Get a wallet receive address")
+@wallet_cmd.command("get_address", short_help="Get the current public wallet address")
 @click.option(
     "-wp",
     "--wallet-rpc-port",
     help="Set the port where the Wallet is hosting the RPC interface. See the rpc_port under wallet in config.yaml",
     type=int,
     default=None,
 )
+@click.option( "-n", "--new_address", help="Generate new public wallet address.", type=bool, default=False, is_flag=True, show_default=True)
 @click.option("-i", "--id", help="Id of the wallet to use", type=int, default=1, show_default=True, required=True)
 @click.option("-f", "--fingerprint", help="Set the fingerprint to specify which wallet to use", type=int)
-def get_address_cmd(wallet_rpc_port: int, id, fingerprint: int) -> None:
-    extra_params = {"id": id}
+def get_address_cmd(wallet_rpc_port: int, id, new_address, fingerprint: int) -> None:
+    extra_params = {"id": id, "get_new_address": new_address}
```

Comment:
It's probably best to also pass `get_new_address: bool` here, i.e. adding the `get_` to both the key and the value.

Reply:
Im not sure what you mean, can you explain please?

Reply:
I think they are referring to the dict key being `"get_new_address"` while the parameter name is just `new_address` and would like them to be the same.  I'm mostly just interested in the `: bool` type hint.  `:]`  But, I see the relevance of adding `get_`.

```suggestion
    extra_params = {"id": id, "get_new_address": get_new_address}
```